src/dialogs/more-info/ha-more-info-dialog.ts

```diff
@@ -243,6 +243,10 @@ export class MoreInfoDialog extends LitElement {
     return [
       haStyleDialog,
       css`
+        ha-dialog {
+          --dialog-content-position: static;
```

Comment:
Are we sure about this? Doesn't this mean that if the dialog is taller than the window, we won't be able to scroll ?

Reply:
I tested it and it seemed fine, but yeah they probably had a reason to make it relative...